packages/js/src/plugins/auctionHouseModule/operations/executeSale.ts

```diff
@@ -188,6 +208,14 @@ export const executeSaleBuilder = (
   const { auctionHouse, listing, bid, auctioneerAuthority } = params;
   const { sellerAddress, asset } = listing;
   const { buyerAddress, tokens } = bid;
+  const {
+    hasAuctioneer,
+    isNative,
+    treasuryMint,
+    address,
```

Comment:
For better readability. Same for other files

```suggestion
    address: auctionHouseAddress,
```